srcpkgs/wiremix/template

```diff
@@ -0,0 +1,20 @@
+# Template file for 'wiremix'
+pkgname=wiremix
+version=0.8.0
+revision=1
+build_style=cargo
+hostmakedepends="pkg-config clang"
+makedepends="pipewire-devel libgit2-1.9-devel"
+short_desc="Simple TUI audio mixer for PipeWire"
+maintainer="Dora 'cat' <cat@thenight.club>"
+license="MIT OR Apache-2.0"
+homepage="https://github.com/tsowell/wiremix/"
+changelog="https://github.com/tsowell/wiremix/raw/main/CHANGELOG.md"
+distfiles="https://github.com/tsowell/wiremix/archive/refs/tags/v${version}.tar.gz"
+checksum=db357d21c76809d674024f1094c2ac6ddd2d6866d4b8ae53cbb0620599006e31
+
```

Comment:
export the variable it will look for and remove the dependency on libgit2

```suggestion

export VERGEN_GIT_DESCRIBE="v${version}"

post_patch() {
	cargo rm --build vergen-git2
}

```